### PR TITLE
Updated Docker integration tests.

### PR DESCRIPTION
**Pull request by Pastafarian**
Tried running the docker integration tests after pulling the project down to take a closer look, found out they weren't currently working so thought I'd get them updated and passing again. 

Good to see all the recent work on the project!

Updated definitions.json to make the username and password consistent with the integration tests.

Updated images from microsoft/dotnet:2.2-sdk to mcr.microsoft.com/dotnet/sdk:8.0 and rabbitmq:3-management to rabbitmq:3.12.12-management.

Removed net472 as a Framework as it prevents projects from running in a Linux container. Given netstandard2.0 is supported, it's superfluous.

### CODE REVIEW

File: tests/Serilog.Sinks.RabbitMQ.Tests.Integration/Serilog.Sinks.RabbitMQ.Tests.Integration.csproj [hunk 1]
```diff
@@ -10,11 +10,9 @@
     <LangVersion>latest</LangVersion>
     <ImplicitUsings>enable</ImplicitUsings>
   </PropertyGroup>
-
   <PropertyGroup Condition=" '$(TargetFramework)' == 'net472' ">
-    <DefineConstants>$(DefineConstants);NET_FX</DefineConstants>
+        <DefineConstants>$(DefineConstants);NET_FX</DefineConstants>
```
Comment (ArieGato) [inline tests/Serilog.Sinks.RabbitMQ.Tests.Integration/Serilog.Sinks.RabbitMQ.Tests.Integration.csproj:11]: Indentation is not as it was. I wouldn't expect any change in this file.

File: tests/Serilog.Sinks.RabbitMQ.Tests.Integration/RabbitMQCollection.cs
Comment (ArieGato) [inline tests/Serilog.Sinks.RabbitMQ.Tests.Integration/RabbitMQCollection.cs:1]: What's the purpose of this file? Why do we need it?

File: tests/Serilog.Sinks.RabbitMQ.Tests.Integration/ConfigurationExtensionsFixture.cs
Comment (ArieGato) [inline tests/Serilog.Sinks.RabbitMQ.Tests.Integration/ConfigurationExtensionsFixture.cs:1]: I wouldn't expect any changes in this file either.

**Comment by Pastafarian** [comment]
Yes, it's a bit of a pain, but it's possible by runnng the integration tests from a mono container that supports the .NET Framework, then install .NET 6.0 and 8.0 into that container. 

A bit of a faff it turns out. But we do now have a version that multi-targets and now runs the previously removed config tests. 

PR updated!

**Review by ArieGato — CHANGES_REQUESTED** [review]

**Review by ArieGato — DISMISSED** [review]

**Comment by Pastafarian** [comment]
Added RabbitMQCollection.cs in an attempt to fix tests before your update to main made it redundent, now removed. Reverted changes to Serilog.Sinks.RabbitMQ.Tests.Integration.csproj and ConfigurationExtensionsFixture.cs

**Review by ArieGato — APPROVED** [review]

**Pull request merged**